README.md

```diff
@@ -1,7 +1,10 @@
+![alt text](https://www.devitoproject.org/images/banner-devito.jpg)
+
 # Devito: Fast Finite Difference Computation from Symbolic Specification
 
 ![Build Status](https://travis-ci.org/opesci/devito.svg?branch=master)
 ![Code Coverage](https://codecov.io/gh/opesci/devito/branch/master/graph/badge.svg)
+[![Binder](http://devito.rocks/badge_logo.svg)](http://devito.rocks/v2/gh/opesci/devito/binder)
```

Comment:
Perfect!

Reply:
I would still put a comment that it doesn't always work/is experimental (failed to build every time I tried)